lib/defaultFormatter.js

```diff
@@ -0,0 +1,45 @@
+var chalk = require('chalk');
+var path = require('path');
+
+module.exports = function() {
+  return function(input) {
```

Comment:
Wasn't aware of that smart feature of chalk. Good to know. I'm still thinking that approach (function accepting options returning a function accepting input) makes sense because of the possibility for options in this or other formatters. Right?